config.default.json

```diff
@@ -4,11 +4,53 @@
     "tradeMnemonic": null,
     "treasuryMnemonic": null,
     "discordApiToken": null,
+    "discordClientId": null,
+    "discordClientSecret": null,
     "twitterConsumerKey": null,
     "twitterConsumerSecret": null,
     "twitterAccessToken": null,
     "twitterAccessTokenSecret": null,
     "twitterId": null,
     "twitterWebhookPort": 3002,
-    "serverPort": 3001
+    "serverPort": 3001,
+    "discordRoleChannel": "welcome",
+
+    "discordRoles": {
+        "artist": ":art: Artist",
+        "builder": ":tools: Builder",
+        "designer": ":triangular_ruler: Designer",
+        "writer": ":writing_hand: Writer",
+        "bridgebuilder": ":bridge_at_night: Bridge Builder",
+        "shiller": ":mega: Shiller"
+    },
+    "discordAdminRole": "Webaverse Team",
+    "discordEmbedContent": {
```

Comment:
This should not be a config, it is more and asset or a constant which can be inclined.

I understand the point of making in configurable. But in that case we should choose what is configurable and expose that rather than putting the discord api into the config.

Reply:
Agreed. Have moved this to be dynamic, per server that invites the discord bot.